### PR TITLE
feat(AuditLog): Include Project Rate Limits in Audit Log

### PR DESCRIPTION
**Pull request by LSSangha**
Plans that have the option of project level rate limits now can see the changes made in the audit log.
screen shot 2019-03-06 at 4 51 46 pm

### CODE REVIEW

File: src/sentry/api/endpoints/project_key_details.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def put(self, request, project, key_id):
                 request=request,
                 organization=project.organization,
                 target_object=key.id,
-                event=AuditLogEntryEvent.PROJECTKEY_EDIT,
+                event=AuditLogEntryEvent.PROJECTKEY_EDIT_RATE_LIMIT,
```
Comment (markstory) [inline src/sentry/api/endpoints/project_key_details.py]: Has the rate limit necessarily been edited? It looks like there are multiple pathways through this method that would result in the rate limit not being modified. Perhaps there the new rate limit audit log should be in addition to the project update audit?

Comment (LSSangha) [reply src/sentry/api/endpoints/project_key_details.py]: Can you explain what you mean by the rate limit not being edited? 
Screen Shot 2019-03-08 at 11 58 53 AM

Comment (markstory) [reply src/sentry/api/endpoints/project_key_details.py]: The PUT method supports renaming a key. A simple rename operation would not modify the rate limit and the audit logs would be incorrect.

Comment (markstory) [reply src/sentry/api/endpoints/project_key_details.py]: I'm still not sure that this change is correct. Renaming or deactivating a key will now show up as a rate-limit change. Should the event type decision be moved up to where the changes are done?

If a project key has its ratelimit and name changed in a single request (something the UI doesn't do but the API supports) would we want the rate limit audit log or a general edit audit log or two logs?

**Comment by BYK** [comment]
Closing due to staleness and conflicts.

**Pull request closed without merging**